### ISSUE TITLE
Error when run `examples/attack/attack_camembert.py`

### ISSUE THREAD

**Issue by Opdoop** [QData/TextAttack#354]
I run textattack command line successfully. But run `examples/attack/attack_camembert.py` give this Error:
```
model_wrapper = HuggingFaceSentimentAnalysisPipelineWrapper(pipeline)
TypeError: Can't instantiate abstract class HuggingFaceSentimentAnalysisPipelineWrapper with abstract methods get_grad
```
Could you help me find out where the problem is?

**Comment by Opdoop** [comment]
Seems like the incompatible with transformer pipeline cause this bug. Changed to the simple version, using textfool to attack bert-uncased model in imdb dataset, it works fine. The code snippet is below.
```
from textattack.attack_recipes import TextFoolerJin2019
from textattack.models.wrappers import HuggingFaceModelWrapper
from transformers import AutoTokenizer, AutoModelForSequenceClassification
import textattack
import tensorflow as tf
physical_devices = tf.config.list_physical_devices('GPU')
tf.config.experimental.set_memory_growth(physical_devices[0], True)

original_model = AutoModelForSequenceClassification.from_pretrained("textattack/bert-base-uncased-imdb")
original_tokenizer = AutoTokenizer.from_pretrained("textattack/bert-base-uncased-imdb")

model = HuggingFaceModelWrapper(original_model, original_tokenizer)
# model_wrapper = HuggingFaceSentimentAnalysisPipelineWrapper(pipeline)

recipe = TextFoolerJin2019.build(model)

dataset = textattack.datasets.HuggingFaceDataset(
    "imdb", split="test", shuffle=False
)

for idx, result in enumerate(recipe.attack_dataset(dataset, indices=range(11))):
    print(('x' * 20), f'Result {idx+1}', ('x' * 20))
    print(result.__str__(color_method='ansi'))
    print()
```

**Comment by jinyongyoo** [comment]
Fixed by #381
